Documentation/allow_legacy_mode.md

```diff
@@ -0,0 +1,101 @@
+## Allow-legacy mode
+
+etcd v2.0 uses internal API v2 protocol, which is not compatible with the v1 protocol.
+
+Allow-legacy mode is introduced to help user upgrade v0.4 etcd cluster easily, and ensures cluster has a short unavailable period, which is less than 1 minute in most cases (based on the size of the actual data).
```

Comment:
"to help users upgrade v0.4.x etcd clusters easily, and ensures that clusters have a short unavailable period" 

s/based/depending/